lib/ansiblelint/rules/custom/__init__.py

```diff
@@ -0,0 +1 @@
+"""placeholder to put custom rules under this dir."""
```

Comment:
I'm not sure how I feel about this approach. But (1) a PEP257-compliant docstring should start with a capital letter:
```suggestion
"""A placeholder package for putting custom rules under this dir."""
```

Also (2), putting init into this package will make it not work with native namespace packages installable by pip.